solr/solr-ref-guide/build.gradle

```diff
@@ -67,8 +67,9 @@ dependencies {
     depVer('org.apache.tika:tika-core')
     depVer('org.apache.zookeeper:zookeeper')
 
-    // jekyll dependencies
-    gems 'rubygems:jekyll:4.2.0'
+    // jekyll dependencies: we cannot use jekyll >=4, as this version uses native lib
+    // ("sassc" instead of deprecated "sass") which fails to compile on some platforms
```

Comment:
It might not be a bad idea to add a reference to the issue SOLR-15520 in this comment so anyone later can find the discussion of why it doesn't work?

Reply:
Wanted to do that, but pressed the commit button too early. Will add the reference to that comment.